### PR TITLE
Feed item form with preview

### PR DESCRIPTION
**Pull request by Xpirix**
- #21 
- #22 
- #31 
Please find below a screenshot of the form. It will be improved according to the next issues.

Update-feed-item

### CODE REVIEW

**Comment by timlinux** [comment]
Change test from "No image choosed" to "No image chosen"

**Comment by timlinux** [comment]
Also let's default the start date to now() and the end date to now + 1 month

**Comment by dimasciput** [comment]
@Xpirix please fix the conflicts

**Comment by Xpirix** [comment]
@dimasciput, Fixed them

File: qgisfeedproject/qgisfeed/views.py [hunk 1]
```diff
@@ -177,4 +177,63 @@ def feeds_list(request):
               "form": form,
               "count": count
             }
-    )
\ No newline at end of file
+    )
+
+
+@staff_required
+def feed_entry_add(request):
```
Comment (dimasciput) [inline qgisfeedproject/qgisfeed/views.py:184]: @Xpirix can you update this to use django class based views? I will merge this for now, you can create a new PR

**Review by dimasciput — APPROVED** [review]

File: qgisfeedproject/qgisfeed/views.py (hunk 1 above)
Comment (Xpirix) [reply qgisfeedproject/qgisfeed/views.py:184]: @dimasciput Okay, thanks. I created a new ticket for it.

**Pull request merged**